soc/silabs/silabs_siwx917/siwg917/Kconfig.soc

```diff
@@ -19,3 +19,34 @@ config SOC
 
 config SOC_PART_NUMBER
 	default "SIWG917M111MGTBA" if SOC_PART_NUMBER_SIWG917M111MGTBA
+#Multiple 917 components depends on NWP and on CMSIS OS 2 thus a common default config here makes sense
+config SIWX917_NWP_NETWORK_STACK
+	bool
+	select CMSIS_RTOS_V2
+	select POLL
+	select DYNAMIC_THREAD
+	select THREAD_NAME
+	select THREAD_STACK_INFO
+	select THREAD_MONITOR
+	select INIT_STACKS
+
+if SIWX917_NWP_NETWORK_STACK
+
+# WiseConnect create threads with realtime priority. Default (10kHz) clock tick
+# prevent proper use of the system with these threads.
+config SYS_CLOCK_TICKS_PER_SEC
+	default 1000
```

Comment:
This is not required anymore AFAIK. See https://github.com/SiliconLabsSoftware/zephyr-silabs/pull/30#discussion_r1765045556

Reply:
Have you tried the WiFi without this?
I've tried the wifi sample (without my change) and I got: "Scan request failed".
BT advertising also fails (my change).

I'd put this back until we discover the root cause.

Reply:
Indeed, it seems this statement is still required.
When I removed it, I didn't fully rebuild my application and SYS_CLOCK_TICKS_PER_SEC was still equal to `100`.
I am going to send a PR to fix that.

(Cc: @asmellby)